src/pages/commits/ActiveCommits/BuildVariantCard.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { MockedProvider } from "@apollo/client/testing";
+import {
+  renderWithRouterMatch as render,
+  screen,
+  userEvent,
+  waitFor,
+} from "test_utils";
+import { BuildVariantCard } from "./BuildVariantCard";
+import { injectGlobalDimStyle, removeGlobalDimStyle } from "./utils";
+
+jest.mock("./utils");
+
+describe("buildVariantCard", () => {
+  it("should call the appropriate functions on hover and unhover", async () => {
+    const mockIntersectionObserver = jest.fn((callback) => {
+      callback([
+        {
+          isIntersecting: true,
+        },
+      ]);
+      return {
+        observe: jest.fn(),
+        disconnect: jest.fn(),
+      };
+    });
+
+    // @ts-expect-error
+    window.IntersectionObserver = mockIntersectionObserver;
+    (injectGlobalDimStyle as jest.Mock).mockImplementationOnce(
+      (taskIdentifier: string) => {
+        Promise.resolve(taskIdentifier);
+      }
+    );
+    (removeGlobalDimStyle as jest.Mock).mockImplementationOnce(() => {});
+
+    render(
+      <MockedProvider mocks={[]} addTypename={false}>
+        <BuildVariantCard
+          variant="ubuntu-2204"
+          height={100}
+          buildVariantDisplayName="Ubuntu 22.04"
+          projectIdentifier="testing"
+          versionId="abc"
+          order={1}
+          tasks={tasks}
+        />
+      </MockedProvider>
+    );
+
+    userEvent.hover(screen.queryByDataCy("build-variant-icon-container"));
+    await waitFor(() => {
+      expect(injectGlobalDimStyle).toHaveBeenCalledTimes(1);
+    });
+
+    userEvent.unhover(screen.queryByDataCy("build-variant-icon-container"));
+    await waitFor(() => {
+      expect(removeGlobalDimStyle).toHaveBeenCalledTimes(1);
+    });
+  });
+});
+
+const tasks = [
+  {
+    id: "1",
+    status: "failed",
+    displayName: "One",
+  },
+  {
+    id: "2",
+    status: "succeeded",
+    displayName: "Two",
+  },
+  {
+    id: "3",
+    status: "succeeded",
+    displayName: "Three",
+  },
+];
```

Comment:
I think you might have to update these to say `"success"`:

<img width="487" alt="Screenshot 2023-06-16 at 1 26 58 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/4486e841-7afc-49e8-99a6-0ff43ec66558">